crates/sui-sdk/src/apis.rs

```diff
@@ -559,21 +981,37 @@ impl GovernanceApi {
         Ok(self.api.http.get_stakes(owner).await?)
     }
 
-    /// Return the committee information for the asked `epoch`.
-    /// `epoch`: The epoch of interest. If None, default to the latest epoch
+    /// Return the [SuiCommittee] information for the provided `epoch`, or an error upon failure.
+    ///
+    /// # Arguments
+    ///
+    /// * `epoch` - the known epoch id or `None` for the last epoch
+    ///
+    /// # Examples
+    ///
+    /// ```rust,no_run
+    /// use sui_sdk::SuiClientBuilder;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), anyhow::Error> {     
+    ///     let sui = SuiClientBuilder::default().build_localnet().await?; // local Sui network
+    ///     let committee_info = sui.read_api().get_committee_info(None).await?;
+    ///     Ok(())
+    /// }
+    /// ```
     pub async fn get_committee_info(
         &self,
         epoch: Option<BigInt<u64>>,
     ) -> SuiRpcResult<SuiCommittee> {
         Ok(self.api.http.get_committee_info(epoch).await?)
     }
 
-    /// Return the latest SUI system state object on-chain.
+    /// Return the latest SUI system state object on-chain, or an error upon failure.
```

Comment:
Needs more detail: What do people look at the system state object for (i.e. what's in it?).